aikau/src/main/resources/alfresco/forms/controls/PushButtons.js

```diff
@@ -144,6 +155,30 @@ define(["alfresco/core/CoreWidgetProcessing",
             name: "alfresco/forms/controls/PushButtonsControl",
             config: config
          });
+      },
+
+      /**
+       * Extends the [inherited function]{@link module:alfresco/forms/controls/BaseFormControl#setOptions}
+       * to make use of any configured [noButtonsLabel]{@link module:alfresco/forms/controls/PushButtons#noButtonsLabel}
+       * when an empty options array is provided.
+       * 
+       * @instance
+       * @param {object[]} options An array of the options to be added.
+       * @since 1.0.80
+       */
+      setOptions: function alfresco_forms_controls_BaseFormControl__setOptions(options) {
+         this.inherited(arguments);
+
+         if (options && 
+             options.length === 0 && 
+             this.noButtonsLabel && 
+             this.wrappedWidget)
+         {
+            var span = domConstruct.create("span", {
+               "class": "alfresco-forms-controls-PushButtons__noOptionsLabel"
+            }, this.wrappedWidget.domNode);
+            span.appendChild(document.createTextNode(this.message(this.noButtonsLabel)));
```

Comment:
Using textContent is as safe as createTextNode and is shorter. There's no particular reason to use one over the other, unless you specifically need a handle on the textNode I guess.